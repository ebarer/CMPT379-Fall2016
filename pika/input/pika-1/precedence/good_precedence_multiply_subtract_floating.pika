exec {

  print 6.1 * 7.3 - 5.7, _n_.
  print 1.1 * 3.9 - 1.6, _n_.
  print 3.4 * 5.9 - 5.5, _n_.
  print 6.4 * 4.0 - 7.0, _n_.
  print 2.5 * 3.4 - 7.0, _n_.
  print 9.4 * 7.7 - 3.4, _n_.
  print 6.0 * 3.6 - 1.5, _n_.
  print 5.2 * 7.4 - 2.0, _n_.
  print 2.8 * 9.8 - 7.0, _n_.
  print 1.2 * 9.2 - 8.0, _n_.
  print 7.7 * 9.0 - 3.2, _n_.
  print 2.2 * 2.3 - 7.5, _n_.

  print 9.8 - 5.2 * 2.6, _n_.
  print 5.1 - 2.7 * 6.2, _n_.
  print 4.4 - 4.4 * 6.3, _n_.
  print 1.4 - 9.6 * 6.4, _n_.
  print 2.4 - 5.0 * 2.1, _n_.
  print 3.2 - 7.3 * 7.3, _n_.
  print 1.7 - 9.5 * 7.1, _n_.
  print 1.1 - 7.8 * 3.5, _n_.
  print 4.2 - 8.5 * 2.2, _n_.
  print 6.3 - 9.0 * 7.2, _n_.
  print 6.4 - 6.1 * 2.1, _n_.
  print 3.5 - 3.2 * 9.9, _n_.

  print 3.0 * 1.0 - 8.7 * 6.1, _n_.
  print 2.2 * 4.2 - 7.6 * 9.6, _n_.
  print 2.3 * 9.5 - 3.6 * 5.7, _n_.
  print 9.0 * 8.3 - 4.8 * 8.5, _n_.
  print 4.6 * 3.0 - 9.8 * 8.1, _n_.
  print 9.9 * 7.2 - 8.6 * 3.7, _n_.
  print 4.4 * 8.5 - 4.9 * 7.4, _n_.
  print 6.6 * 4.5 - 4.3 * 6.9, _n_.
  print 8.1 * 1.1 - 6.9 * 1.1, _n_.
  print 8.6 * 7.6 - 7.2 * 5.2, _n_.
  print 8.7 * 2.6 - 2.0 * 8.5, _n_.
  print 2.4 * 4.4 - 3.0 * 2.5, _n_.

  print 8.4 - 7.9 * 1.0 - 9.6, _n_.
  print 7.6 - 1.2 * 9.5 - 1.7, _n_.
  print 3.8 - 1.7 * 5.6 - 9.5, _n_.
  print 4.1 - 9.8 * 7.1 - 2.9, _n_.
  print 2.6 - 1.5 * 4.0 - 3.2, _n_.
  print 8.5 - 2.6 * 1.9 - 7.4, _n_.
  print 2.4 - 4.8 * 8.5 - 3.4, _n_.
  print 4.3 - 5.0 * 2.4 - 8.5, _n_.
  print 4.8 - 8.0 * 9.3 - 9.6, _n_.
  print 5.9 - 5.7 * 4.8 - 5.5, _n_.
  print 2.1 - 5.1 * 5.7 - 3.1, _n_.
  print 7.9 - 7.7 * 6.1 - 4.9, _n_.

}

