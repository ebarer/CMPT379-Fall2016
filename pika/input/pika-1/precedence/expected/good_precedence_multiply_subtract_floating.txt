38.83
2.69
14.56
18.6
1.5
68.98
20.1
36.48
20.44
3.04
66.1
-2.44
-3.72
-11.64
-23.32
-60.04
-8.1
-50.09
-65.75
-26.2
-14.5
-58.5
-6.41
-28.18
-50.07
-63.72
1.33
33.9
-65.58
39.46
1.14
0.03
1.32
27.92
5.62
3.06
-9.1
-5.5
-15.22
-68.38
-6.6
-3.84
-41.8
-16.2
-79.2
-26.96
-30.07
-43.97
